### ISSUE TITLE
[Feature Request]: Don't rely too heavily on telepathy accounts being needed

### ISSUE THREAD

**Issue by sagetarian** [sagetarian/skype-wrapper#12]
at the moment skype-wrapper tells a telepathy account to change its status .. so you need a working telepathy account set up ..

**Comment by hdave** [comment]
I second this, but I think this is an issue more with Ubuntu.  I uninstalled Empathy (as I use Pidgin) and once I did that I could no longer change my status in my messaging menu, even though skype-wrapper is integrated with it.  Pigdin is not integrated with it.   In any event, I had to reinstalled Empathy AND give myself a non-IRC chat account before Ubuntu would let me set my status.  I have created an issue for this in launchpad.  Please go and vote for it:

https://bugs.launchpad.net/ubuntu/+source/unity/+bug/994607

**Comment by jnv** [comment]
I agree that skype-wrapper shouldn't rely on Telepathy/Mission Control to change status.
However I am pretty sure that Pidgin supports status change through messaging menu – it's a separate package called `indicator-status-provider-pidgin`. It's part of the [indicator-messages](https://launchpad.net/indicator-messages) project which is probably the best place to look for reference implementations.

**Comment by sagetarian** [comment]
thank you jnv .. having a look at that code, i should be able to try make a indicator-messages service provider for skype-wrapper .. i've been looking for this for some time :D you are a genius
